Sources/TokamakCore/Tokens/Color.swift

```diff
@@ -94,10 +103,10 @@ public class _ConcreteColorBox: AnyColorBox {
 public class _EnvironmentDependentColorBox: AnyColorBox {
   public let resolver: (EnvironmentValues) -> Color
 
-  public static func == (lhs: _EnvironmentDependentColorBox,
-                         rhs: _EnvironmentDependentColorBox) -> Bool
-  {
-    lhs.resolver(EnvironmentValues()) == rhs.resolver(EnvironmentValues())
+  override public func equals(_ other: AnyColorBox) -> Bool {
```

Comment:
What's the reason for using `equals` instead of `==` and `Equatable`? It should be added in a corresponding code comment.